tests/ip/test_mgmt_ipv6_only.py

```diff
@@ -58,3 +58,23 @@ def test_show_features_ipv6_only(duthosts, enum_dut_hostname, convert_and_restor
                                     .format(cmd_key), module_ignore_errors=False)['stdout']
         pytest_assert(redis_value.lower() == cmd_value.lower(),
                       "'{}' is '{}' which does not match with config_db".format(cmd_key, cmd_value))
+
+
+def test_image_download_ipv6_only(creds, duthosts, enum_dut_hostname,
+                                  convert_and_restore_config_db_to_ipv6_only): # noqa F811
+    """
+    Test image download in mgmt ipv6 only scenario
+    """
+    duthost = duthosts[enum_dut_hostname]
+    image_url = creds.get("test_image_url", {}).get("ipv6", "")
+    pytest_require(len(image_url) != 0, "Cannot get image url")
```

Comment:
In future,
We can start a simple http server in PTF. Then no dependency on image server url.